orchestrator/dags/general_data_import.py

```diff
@@ -243,7 +243,7 @@ def extract_routes_lines(ti):
     ti.xcom_push(key="routes_output_file", value=routes_output_file)
 
 
-def create_routes_json(input_file):
+def create_routes_json(input_file: str) -> list:
```

Comment:
Similar as above. But here we can specify returned type in two ways:
 * easy: `List[Dict]`
 * more specific, where we specify the entire dict structure: `List[Dict[str, Union[str, int, List[Dict[str, Union[str, int, List[Dict[str, Union[str, int, bool]]]]]]]]]` - here I'm absolutely not sure if I typed it correctly :sweat_smile:. so if you want to use it, please review it first